storage/innobase/log/log0write.cc

```diff
@@ -2667,6 +2667,11 @@ void log_write_notifier(log_t *log_ptr) {
       log.write_notifier_resume_lsn.store(0, std::memory_order_release);
 
       log_write_notifier_mutex_enter(log);
+    } else if (UNIV_UNLIKELY(log.write_notifier_resume_lsn.load(
```

Comment:
@ldonoso Luis, It took me a few hours to understand the race conditions here :)
However, I am not sure this change is 100%. safe.
Something tells me that the proper fix would be to add something similar to what Oracle did with the `while` construct in the `log_resume_writer_threads ()` function
https://github.com/mysql/mysql-server/blob/98b2ccb470de120d36bc4a623c814cdfded958ec/storage/innobase/log/log0log.cc#L909
to the end of `log_pause_writer_threads()`.
Basically,  we need to guarantee that at the end of `log_pause_writer_threads()` both `log_write_notifier` and `log_flush_notifier` threads are waiting on `writer_threads_resume_event` here
https://github.com/mysql/mysql-server/blob/98b2ccb470de120d36bc4a623c814cdfded958ec/storage/innobase/log/log0write.cc#L2582
and here
https://github.com/mysql/mysql-server/blob/98b2ccb470de120d36bc4a623c814cdfded958ec/storage/innobase/log/log0write.cc#L2700